docs/estela/installation/micro-frontends.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: Micro-Frontends Guide
+parent: Installation
+grand_parent: estela
+---
+
+# estela Micro-Frontends Guide (Optional)
+
+estela allows the use of Micro-Frontends only in the user dropdown. This is achieved with the use of [Module Federation](https://webpack.js.org/concepts/module-federation/).
+
+## Requirements
+- The micro-frontends must use [Module Federation](https://webpack.js.org/concepts/module-federation/)
+- The micro-frontends must have and expose their routes with the name of ComponentRoutes
+    ```js
+    // example
```

Comment:
A real example would be more descriptive, do we have a demo micro frontend? The full pipeline should be described, it could be another PR